1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -1,3 +1,17 @@
+
+
+let btn = document.querySelector("#Qbtn");
```

Comment:
The space on the button's id breaks this, but it works if you fix that, and it's a simple mistake so  good work!